app/build.gradle

```diff
@@ -25,6 +25,11 @@ android {
         buildConfig true
         viewBinding true
     }
+    bundle {
```

Comment:
@andhikayuana  is this a complex concept. Not familiar with it. Could you briefly explain ?

Reply:
this is related with android app bundle concept. in short, when the users download the app from playstore it will only download the specific architecture, specific language
more info https://developer.android.com/guide/app-bundle/configure-base#disable_config_apks